site/en/docs/web-platform/navigation-api/index.md

```diff
@@ -277,27 +256,27 @@ While for many sites the most common case will be when the user clicks a `<a hre
 
 First is programmatic navigation, where navigation is caused by a method call inside your client-side code.
 
-You can call `appHistory.navigate('/another_page')` from anywhere in your code to cause a navigation.
-This will be handled by the centralized event handler registered on the "navigate" handler, and your centralized handler will be called synchronously.
+You can call `navigation.navigate('/another_page')` from anywhere in your code to cause a navigation.
+This will be handled by the centralized event listener registered on the "navigate" listener, and your centralized listener will be called synchronously.
 
-This is intended as an improved aggregation of older methods like `location.assign()` and friends, plus the History API's methods `pushState()` and `replacestate()`.
+This is intended as an improved aggregation of older methods like `location.assign()` and friends, plus the History API's methods `pushState()` and `replaceState()`.
 
 {% Aside %}
 
-These older programmatic methods for changing the URL are all still supported with the App History API and now fire the "navigate" handler.
+These older programmatic methods for changing the URL are all still supported with the Navigation API and now fire the "navigate" listener.
 That is, they're also handled centrally.
 Their signatures aren't modified in any way (i.e., they won't now return a `Promise`) by this new specification, and we imagine that in an older codebase, they'll be replaced by calls to `.navigate()` over time.
 
 {% endAside %}
 
-The `AppHistory.navigate()` method returns a object which contains two `Promise` instances in `{ committed, finished }`.
-This allows the invoker can wait until either the transition is "committed" (the visible URL has changed and a new `AppHistoryEntry` is available) or "finished" (all promises passed to `transitionWhile()` are complete&mdash;or rejected, due to failure or being preempted by another navigation).
+The `navigation.navigate()` method returns a object which contains two `Promise` instances in `{ committed, finished }`.
+This allows the invoker can wait until either the transition is "committed" (the visible URL has changed and a new `NavigationEntry` is available) or "finished" (all promises passed to `transitionWhile()` are complete&mdash;or rejected, due to failure or being preempted by another navigation).
 
 The `navigate` method also has an optional options object which controls how the navigation will occur.
-These options will allow you to `replace` the current URL, set a new immutable `state` (to be made available via `AppHistoryEntry.getState()`), and configure `AppHistoryNavigateEvent.info`.
+These options will allow you to `replace` the current URL, set a new immutable `state` (to be made available via the `.getState()` method on `NavigationEntry`s), and configure `navigateEvent.info`.
```

Comment:
`replace` option got converted into `history: "replace"`, so this probably needs slight rephrasing.